play-media-native/components/Listing/Listing.tsx

```diff
@@ -115,6 +115,9 @@ export const Listing = ({
           />
         }
         onScroll={onScroll}
+        removeClippedSubviews
```

Comment:
I read a bit and it seems removeClippedSubviews has an issue when used with tabs on iOS. Please add this prop only on android devices. I don't know if it's going to lower performance, but I don't think there's a fix for it.

Reply:
Removing it altogether as it does not offer any huge performance improvement